eng/common/Extract-ReleaseNotes.ps1

```diff
@@ -1,9 +1,12 @@
 # given a CHANGELOG.md file, extract the relevant info we need to decorate a release
 param (
   [Parameter(Mandatory = $true)]
-  [String]$ChangeLogLocation
+  [String]$ChangeLogLocation,
+  [String]$VersionString
 )
 
+$ErrorActionPreference = 'Stop'
```

Comment:
What prompted you to set this ErrorActionPreference?

Reply:
Since its outputting to console and capturing output I added this to make sure the it doesn't continue running after some kind of error has occurred.